action.yml

```diff
@@ -62,7 +62,7 @@ runs:
 
     - name: Upload OCI distribution-spec conformance results as build artifact
       if: always() && steps.run-conformance.outputs.has-report == 'true'
-      uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+      uses: actions/upload-artifact@65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08 # v4.6.0
```

Comment:
for future update: would be nice to uses hashes for all of them, then dependabot can do these updates for us

Reply:
I'm happy to make that change. Just need to ensure dependabot is setup (it hasn't been updating this file).

Reply:
Yep I don't see a dependabot on this repo, a basic setup would be https://github.com/Trow-Registry/trow/blob/5b794f588ba0e1a6b7d57b6a25f9e6c4724b0b62/.github/dependabot.yml